Doc/whatsnew/3.15.rst

```diff
@@ -785,6 +785,14 @@ New features
   (Contributed by Victor Stinner in :gh:`129813`.)
 
 
+Limited C API changes
```

Comment:
`Py_TPFLAGS_MANAGED_DICT` and `Py_TPFLAGS_MANAGED_WEAKREF` are not limited C API, right?